editor/src/input/input_mapper.rs

```diff
@@ -133,7 +134,7 @@ impl Default for Mapping {
 		// it as an available action in the respective message handler file (such as the bottom of `document_message_handler.rs`)
 		let mappings = mapping![
 			// Higher priority than entries in sections below
-			entry! {action=DocumentsMessage::Paste, key_down=KeyV, modifiers=[KeyControl]},
+			entry! {action=DocumentsMessage::Paste(User), key_down=KeyV, modifiers=[KeyControl]},
```

Comment:
Where does the `User` symbol come from? I don't see it above in the file.

Reply:
Thats part of `use Clipboard::*`